### PR TITLE
depot writer can set the recipe on the request depot using the first …

### PR DESCRIPTION
**Pull request by totobest**
…signal which is a valid recipe

### CODE REVIEW

File: script/depots/request_depot.lua [hunk 1]
```diff
@@ -404,8 +404,8 @@ function request_depot:update_circuit_writer()
   local circuit_condition = behavior.connect_to_logistic_network and behavior.logistic_condition or behavior.circuit_condition
   if circuit_condition then
     local condition = circuit_condition.condition
-    if condition.comparator == "=" then
-      local first_signal = condition.first_signal
+    local first_signal = condition.first_signal
+    --[[ if condition.comparator == "=" then
```
Comment (Klonan) [inline script/depots/request_depot.lua:408]: Why are you commenting this out?

Comment (totobest) [reply script/depots/request_depot.lua:408]: It is not documented and when I've found this code, I thought it is not an obvious behavior.

Comment (Klonan) [reply script/depots/request_depot.lua:408]: It is documented int he in-game tips and tricks:
https://github.com/Klonan/Transport_Drones/blob/master/locale/en/transport_drones.cfg#L71

Comment (totobest) [reply script/depots/request_depot.lua:408]: My bad, I'll make another PR when I have the time.

**Pull request closed without merging**